nbgitpuller/handlers.py

```diff
@@ -192,7 +192,7 @@ class LegacyInteractRedirectHandler(IPythonHandler):
     @gen.coroutine
     def get(self):
         repo = self.get_argument('repo')
-        account = self.get_argument('account', 'data-8')
```

Comment:
Or we could just remove this line and merge it now :D

Reply:
I've added it back